src/objects.js

```diff
@@ -1,42 +1,57 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+}
 };
 
-const getName = object => {
-  // your code here
-};
+const getName = object => object.name
 
-const getProperty = (property, object) => {
-  // your code here
-};
 
-const hasProperty = (property, object) => {
-  // your code here
-};
+const getProperty = (property, object) => (object[property])
+  // return {
+  //   property.age,
+  //   property.name,
+
+
+const hasProperty = (property, object) =>
+  object.hasOwnProperty(property);
```

Comment:
You can fix this by accessing the prototype property of the object as follows:
```
  return object.prototype.hasOwnProperty.call(property);
```

When eslint throws an error, hover over it and there will be a link under quick fix to show documentation for the rule being violated, this will almost always show you how to fix it and give you a cleaner more robust codebase as a result